font/fonts.css

```diff
@@ -0,0 +1,21 @@
+@font-face {
+  font-family: 'Gotham Pro';
+  src:
+    local('Gotham Pro Bold'),
+    local('GothamPro-Bold'),
+    url('GothamPro-Bold.woff2') format('woff2'),
+    url('GothamPro-Bold.woff') format('woff');
+  font-weight: bold;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'Gotham Pro';
+  src:
+    local('Gotham Pro'),
+    local('GothamPro'),
+    url('GothamPro.woff2') format('woff2'),
+    url('GothamPro.woff') format('woff');
+  font-weight: normal;
+  font-style: normal;
+}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/Mandri_Firsttask/pullRequest?prid=5484031)